tests/e2e/specs/in-game/basicMoves.spec.js

```diff
@@ -13,7 +9,7 @@ describe('Game Basic Moves - P0 Perspective', () => {
 
   it('Plays Points', () => {
     // Set Up
-    cy.loadGameFixture({
+    cy.loadGameFixture(0, {
```

Comment:
I believe we can remove the two lines after this function call (like you've done elsewhere):

```
// these are now handled by the command itself
    cy.get('[data-player-hand-card]').should('have.length', 2);
    cy.log('Loaded fixture');
```